### PR TITLE
feat: add horizontal stepper component

### LINKED ISSUE #14: Horizontal Stepper
Stepper component to show steps and current step state.

### PR DESCRIPTION
**Pull request by tkmct**
Added horizontal stepper component
close #14

### CODE REVIEW

File: packages/kit/src/Stepper.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react'
+import './stepper.css'
+import UIContext from './contexts/interface'
+
+export default ({ maxStep = 3, currentStep = 1, size, ...props }) => {
+  const ui = useContext(UIContext)
+  return (
+    <div className={`stepper-container ${ui.modeCssClass} ${size ?? 'normal'}`} {...props}>
+      {[...Array(maxStep).keys()].map((v) => (
```
Comment (vimwitch) [inline packages/kit/src/Stepper.js:9]: This seems complicated, could just do `Array(maxStep).fill().map((v) =>`

File: packages/kit/src/stepper.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  background-color: var(--color-gray-2);
+}
+
+.stepper-container.dark > .stepper-item.active {
+  color: var(--color-black);
+  background-color: var(--color-white);
+}
+
+.stepper-container.dark > .stepper-item.inactive {
+  color: var(--color-gray-2);
```
Comment (vimwitch) [inline packages/kit/src/stepper.css:43]: Maybe make this a bit lighter? It's sorta hard to read in darkmode. Maybe color: white and background: gray 6? @ChialiTsai 

image

**Review by vimwitch — CHANGES_REQUESTED** [review]
Looks sick!

**Comment by vimwitch** [comment]
It would be cool if each step could have a description. Maybe shown on mouseover or as text beneath the number/circle.

@ChialiTsai What do you think of descriptions for steps?

File: packages/kit/src/Stepper.js (hunk 1 above)
Comment (tkmct) [reply packages/kit/src/Stepper.js:9]: Thank you, didn't know that!

**Comment by vimwitch** [comment]
Changes look good to me

**Review by vimwitch — APPROVED** [review]

**Pull request merged**